en/sensor/lidar_lite.md

```diff
@@ -0,0 +1,49 @@
+# Lidar-Lite
```

Comment:
FYI, I know this is a bit "thin" - it just copies docs from pixhawk.org. We really need update that shows I2C connections and power connections in both cases. 
needed because of old link to these docs in user guide.